### PR TITLE
fix: message removed when not logged in

### LINKED ISSUE #43401: (client): Remove sentence from donor success message that doesn't apply
After making a donation, you get this success message whether you are signed in or not:
Screen Shot 2021-09-07 at 9 14 31 PM

I don't think that last sentence should show up if you aren't signed in. Doesn't really make sense.

### PR DESCRIPTION
**Pull request by EhEhRon91**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #43401

Tested by adding the component on donation page and changing boolean value in the code directly. If there are any issues, let me know.

### CODE REVIEW

**Comment by EhEhRon91** [comment]
Did a ``rebase -i main`` then ``edit`` on the commit to change the message and add back the spacing of the imports that was changed.

File: client/src/components/Donation/DonateForm.tsx [hunk 1]
```diff
@@ -328,7 +330,13 @@ class DonateForm extends Component<DonateFormProps, DonateFromComponentState> {
...
     const {
-      donationFormState: { processing, success, error, redirecting },
+      donationFormState: {
+        processing,
+        success,
+        error,
+        redirecting,
+        isSignedIn
+      },
       isMinimalForm
```
Comment (ahmadabdolsaheb) [inline client/src/components/Donation/DonateForm.tsx:340]: isSignedin is already defined as a prop so we don't need to include it in the donationFromState.
You could destruct it similar to isMinimalForm here and pass it to renderCompletion.

**Review by hamzawaleed0102 — APPROVED** [review]
LGTM, Approved!

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by ahmadabdolsaheb** [comment]
Thanks @EhEhRon91 for this fix.

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**